core/src/main/java/org/apache/druid/java/util/common/parsers/JSONFlattenerMaker.java

```diff
@@ -108,6 +108,22 @@ public Object getRootField(final JsonNode obj, final String key)
     }
   }
 
+  @Override
+  public Function<JsonNode, Object> makeJsonTreeExtractor(final List<String> exprs)
+  {
+    String[] exprsArray = exprs.toArray(new String[0]);
```

Comment:
Why do we need to get the string array first? Can't we use the `exprs` object directly in the loop below?

Reply:
This is a defensive copy.